packages/pf4-component-mapper/src/files/field-array.d.ts

```diff
@@ -0,0 +1,16 @@
+import { ReactNode } from "react";
+import { Field } from "@data-driven-forms/react-form-renderer";
+
+export interface FieldArrayProps {
+  label?: ReactNode;
+  description?: ReactNode;
+  fields: Field[];
```

Comment:
Oh.. does field requires `name`? FieldArray field if there is only one, does not require the name.

Reply:
yeah name is required always in `Field` type

Reply:
So then it should not be field :D

Reply:
@rvsia I've added special type for the field array field